src/Symfony/Component/TypeInfo/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/type-info",
+    "type": "library",
+    "description": "Extracts information about PHP types using metadata of popular sources",
+    "keywords": [
+        "type",
+        "phpdoc",
+        "phpstan",
+        "symfony"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Mathias Arlaud",
+            "email": "mathias.arlaud@gmail.com"
+        },
+        {
+            "name": "Baptiste LEDUC",
+            "email": "baptiste.leduc@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2"
+    },
+    "require-dev": {
+        "symfony/dependency-injection": "^7.1",
+        "phpdocumentor/type-resolver": "^1.7",
+        "phpstan/phpdoc-parser": "^1.0"
```

Comment:
Should these be added to `suggests` or even `require`? Or am I missing something.

Reply:
We don't add composer' `suggests` anymore as it appears to not be that helpful, mostly noise that very few people read